backend/open_webui/utils/middleware.py

```
@@ -740,9 +760,12 @@ async def process_chat_payload(request, form_data, user, metadata, model):
 
     system_message = get_system_message(form_data.get("messages", []))
     if system_message:
-        form_data = apply_system_prompt_to_body(
-            system_message.get("content"), form_data, metadata, user
-        )
+        try:
+            form_data = apply_system_prompt_to_body(
+                system_message.get("content"), form_data, metadata, user
+            )
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)